.yamato/promotion.yml

```diff
@@ -31,17 +32,6 @@ promotion_test_{{ platform.name }}_{{ editor.version }}:
 
 promotion_test_trigger:
   name: Promotion Tests Trigger
-  agent:
```

Comment:
It is not required to define an agent? 
Do the Package Promotion tests still work without these lines?

Reply:
Because this job doesn't execute any commands, it doesn't need an agent. More info on Virtual Jobs here: https://internaldocs.hq.unity3d.com/yamato/yaml/job-definitions/#virtual-jobs

Reply:
Thanks for the link!